WooCommerce/Classes/Tools/Zendesk/ZendeskManager.swift

```diff
@@ -231,16 +255,34 @@ final class ZendeskManager: NSObject, ZendeskManagerProtocol {
 
     /// Returns the user's Support email address.
     ///
-    func userSupportEmail() -> String? {
-        let _ = getUserProfile()
-        return userEmail
+    func userSupportIdentity() -> (name: String?, emailAddress: String?) {
+        if getUserProfile() {
+            return (userName, userEmail)
+        }
+        getUserInformationIfAvailable()
+        return (userName, userEmail)
     }
 }
 
 // MARK: - Private Extension
 //
 private extension ZendeskManager {
 
+    @MainActor
+    func createZendeskIdentity() async throws {
+        return try await withCheckedThrowingContinuation { continuation in
+            createZendeskIdentity { [weak self] success in
+                guard let self, success else {
+                    DDLogInfo("Creating Zendesk identity failed.")
```

Comment:
nit❓ Should this be `DDLogError`?

Reply:
Updated in 3871ce9743e5a2e3612d08f6d421b19c253a4560.